### ISSUE TITLE
I use `--gpu 1.0` to train new model, but when I enter `nvidia-smi` there is no process use GPU

### ISSUE THREAD

**Issue by lizekui** [thtrieu/darkflow#367]
My command is :

`./flow --model cfg/tiny-yolo-voc-3c.cfg --load bin/tiny-yolo-voc.weights --train --annotation VOC2007/Annotations --dataset VOC2007/JPEGImages/ --gpu 1.0 --epoch 15000 --save 1000`

I use `--gpu 1.0` but when I enter `nvidia-smi` there is no process use GPU :

> No running processes found

and the console log is (well log is `GPU mode with 1.0 usage`):

> Parsing ./cfg/tiny-yolo-voc.cfg
Parsing cfg/tiny-yolo-voc-3c.cfg
Loading bin/tiny-yolo-voc.weights ...
Successfully identified 63471556 bytes
Finished in 0.008124589920043945s

> Building net ...
Source | Train? | Layer description                | Output size
-------+--------+----------------------------------+---------------
       |        | input                            | (?, 416, 416, 3)
 Load  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 416, 416, 16)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 208, 208, 16)
 Load  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 208, 208, 32)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 104, 104, 32)
 Load  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 104, 104, 64)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 52, 52, 64)
 Load  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 52, 52, 128)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 26, 26, 128)
 Load  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 26, 26, 256)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 13, 13, 256)
 Load  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 512)
 Load  |  Yep!  | maxp 2x2p0_1                     | (?, 13, 13, 512)
 Load  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 1024)
 Load  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 13, 13, 1024)
 Init  |  Yep!  | conv 1x1p0_1    linear           | (?, 13, 13, 40)
-------+--------+----------------------------------+---------------
**GPU mode with 1.0 usage**
cfg/tiny-yolo-voc-3c.cfg loss hyper-parameters:
	H       = 13
	W       = 13
	box     = 5
	classes = 3
	scales  = [1.0, 5.0, 1.0, 1.0]
Building cfg/tiny-yolo-voc-3c.cfg loss
Building cfg/tiny-yolo-voc-3c.cfg train op
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE3 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
Finished in 13.823598861694336s

> Enter training ...

> cfg/tiny-yolo-voc-3c.cfg parsing VOC2007/Annotations
Parsing for ['Reactionary-1', 'Reactionary-2', 'Reactionary-3'] 
E tensorflow/core/util/events_writer.cc:62] Could not open events file: ./summary/train/events.out.tfevents.1501880729.centos7: Not found: ./summary/train/events.out.tfevents.1501880729.centos7
E tensorflow/core/util/events_writer.cc:95] Write failed because file could not be opened.
[====================>]100%  000511.xml
Statistics:
Reactionary-1: 368
Reactionary-2: 216
Reactionary-3: 177
Dataset size: 511
Dataset of 511 instance(s)
Training statistics: 
	Learning rate : 1e-05
	Batch size    : 16
	Epoch number  : 15000
	Backup every  : 1000
step 1 - loss 113.30760192871094 - moving ave loss 113.30760192871094
step 2 - loss 112.68678283691406 - moving ave loss 113.24552001953126
step 3 - loss 111.12446594238281 - moving ave loss 113.03341461181641
blarblar....

So what problem during my training ? Thanks all~

**Comment by jubjamie** [comment]
Please can you give us some system specs and then answer the following.

1.  What version of Tensorflow are you running and what type? CPU or GPU?
2.  Please post the output of nvidia-smi before you start the program and then run nvidia-smi **again** once you are during a step (e.g. step 3).

Thanks

**Comment by lizekui** [comment]
Oh my dear @jubjamie , nice to meet you again, thanks for attention !

My tf version is 1.0.0 ;
My CPU is Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz with 16 Kernels
My GPU is Quadro K5000 (poor performance cause my PC is for video rendering... hah)

AND the nvidia-smi output before I run my code :

> (tf100) zkli@centos7:~/repo/darkflow/zklitest-flag$ nvidia-smi 
Sat Aug  5 00:31:31 2017       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 375.26                            Driver Version: 375.26                    |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|  Memory-Usage | GPU-Util  Compute M. |
|==============+================+===============|
|   0  Quadro K5000        Off  | 0000:06:00.0     Off |                  Off |
| 30%   43C    P0    32W / 137W |      0MiB /  4035MiB |      1%      Default |
+-------------------------------+----------------------+----------------------+
                                                                               

> +-----------------------------------------------------------------------------+

> | Processes:                                                                       GPU Memory |
|  GPU       PID  Type  Process name                                           Usage      |
|==============================================|
|  No running processes found                                                                   |
+-----------------------------------------------------------------------------+

after running my code :

> (tf100) zkli@centos7:~/repo/darkflow/zklitest-flag$ nvidia-smi 
Sat Aug  5 00:36:28 2017       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 375.26                            Driver Version: 375.26                    |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap| Memory-Usage | GPU-Util  Compute M. |
|===============+==============+=================|
|   0  Quadro K5000        Off  | 0000:06:00.0            Off |                          Off |
| 30%   42C    P0    32W / 137W |      0MiB /  4035MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
                                                                               
> +-----------------------------------------------------------------------------+
| Processes:                                                                              GPU Memory |
|  GPU       PID  Type  Process name                                               Usage      |
================================================|
|  No running processes found                                                                     |
+-----------------------------------------------------------------------------+

indeed nothing new found between output above...

**Comment by leoxiao2012** [comment]
@lizekui It's clear that you were using tensorflow with cpu only. you can follow the official guide, and reinstall the gpu version tensorflow which requires CUDA and cuDNN.

**Comment by lizekui** [comment]
Thanks a lot ! @leoxiao2012  Too unfortunately ... but I indeed meet GPU is running when I run another  long time ago ... I am ready to reinstall my CUDA and cuDNN haha

**Comment by lizekui** [comment]
Finally I reinstall CUDA and cuDNN , but still not work ... 
I found tf is Anaconda plain tensorflow version, not another version named "tensorflow-gpu",
I install tensorflow-gpu according to below command, and it is work :

`conda install tensorflow-gpu=1.0.1`

Thanks @jubjamie @leoxiao2012

**Comment by jubjamie** [comment]
To get the most up to date features I suggest that you replace your conda installation with an official pip installation. You can do this by uninstalling your conda version and installing with `pip install tensorflow-gpu`

**Comment by ahkarami** [comment]
Dear @jubjamie,
I have installed TefnsorFlow-GPU version 1.7 with CUDA 9.0 & cuDNN 7. But I have problem to run darkflow. Is there any problem with _darkflow_ & TensorFlow 1.7?
